### PR TITLE
Adding option to set file download directory.

### PR DESCRIPTION
**Pull request by arbrown**
Exposed a property on WebInterface.cs that would allow users to specify a directory to download and extract zip files into.  Allows users who do not have necessarily have access to their application default directory (like ASP.NET MVC sites running locally) to specify a different directory to use.

This comes after [my previous PR](https://github.com/VDSoft/TheTVDBApi/pull/4) but could be separated into a separate PR if necessary.

### CODE REVIEW

File: src/TVDB/Web/WebInterface.cs [hunk 1]
```diff
@@ -593,7 +602,7 @@ public WebInterface(string apiKey)
 			{
 				foreach (ZipArchiveEntry entry in archive.Entries)
 				{
-					entry.ExtractToFile(Path.Combine(dirInfo.Name, entry.Name), true);
+					entry.ExtractToFile(Path.Combine(dirInfo.FullName, entry.Name), true);
```
Comment (ChristophvdF) [inline src/TVDB/Web/WebInterface.cs:605]: Using the FullName property could be a bit nasty because it will create all containing sub folders of the zip file and we don't know if the developer of TheTVDB API will change the packing behavior. My intention was to create a flat folder which contains all data regardless of the original struct of the zip file.

**Comment by arbrown** [comment]
Actually it won't create any subfolders.  The `dirInfo.FullName` call will only get the full name of the directory you have access to write into.  Then it still writes the file from the zip using only `entry.Name` instead of `entry.FullName.`  So, the behavior is exactly the same if you leave `FileDirectory` empty, and will only change the location of the "loaded.zip" and the "extraction" folder if you specify a directory, but will not change the contents or layout of the file/folder.

Basically, I was trying to use this library in an ASP MVC website, and it was trying to download the zip file to {Application Root}/loaded.zip which ended up being in Program Files and the IIS directory, and the app didn't have the ability to write there.  I was able to change the FileDirectory to another directory where I _did_ have write permissions, and it saved loaded.zip there and extracted into a flat file directory just in a different directory.
